lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -63,8 +63,9 @@ spec = do
             , separator = Nothing
             , input = "patate\n14\n"
             , expectedStdout =
-                "Prompt: ******\n\ESC[91minput does not \
-                \start with a digit\n\ESC[mPrompt: **\n"
+                "Prompt: ******\n\ESC[91mInt is an \
+                \integer number between -9223372036854775808 \
+                \and 9223372036854775807.\n\ESC[mPrompt: **\n"
```

Comment:
The documentation for `Data.Int` states:
> A fixed-precision integer type with at least the range [-2^29 .. 2^29-1]

This implies that the range can actually be **wider** than stated above, depending on the Haskell implementation.

In light of this, perhaps it is better to use `minBound` and `maxBound` to generate the above string.

Alternatively, we could stop using `Int` within our public APIs, and instead use fixed sized values (such as `Int32`, `Int64`) or unbounded values (`Integer`).